fillers/eips/eip4844/excess_data_gas.py

```diff
@@ -772,15 +772,12 @@ def test_invalid_blob_txs(fork: Fork):
                 tx_count=MAX_BLOBS_PER_BLOCK + 1,
                 blobs_per_tx=1,
             ),
-            # TODO: Enable, at the time of writing this test case, the EIP
-            # does not specify a minimum blob amount for the type 3
-            # transaction.
-            # InvalidBlobTransactionTestCase(
-            #     tag="blob_underflow",
-            #     parent_excess_blobs=10,  # data_gasprice= 1
-            #     tx_error="too_few_blobs",
-            #     blobs_per_tx=0,
-            # ),
+            InvalidBlobTransactionTestCase(
```

Comment:
I think we should merge #110 first if the change is required for this PR.

Reply:
Sounds good!